### PR TITLE
Remove some DeprecatedString uses in file.cpp

### PR DESCRIPTION
**Pull request by nico**

### CODE REVIEW

File: Userland/Utilities/file.cpp [hunk 1]
```diff
@@ -104,48 +104,48 @@ static Optional<DeprecatedString> elf_details(DeprecatedString description, Depr
...
+    __ENUMERATE_MIME_TYPE_DESCRIPTION("image/png"sv, "PNG image data", image_details)                                 \
+    __ENUMERATE_MIME_TYPE_DESCRIPTION("image/webp"sv, "WebP image data", image_details)                               \
+    __ENUMERATE_MIME_TYPE_DESCRIPTION("image/x-portable-bitmap"sv, "PBM image data", image_details)                   \
+    __ENUMERATE_MIME_TYPE_DESCRIPTION("image/x-portable-graymap"sv, "PGM image data", image_details)                  \
+    __ENUMERATE_MIME_TYPE_DESCRIPTION("image/x-portable-pixmap"sv, "PPM image data", image_details)                   \
+    __ENUMERATE_MIME_TYPE_DESCRIPTION("image/x-qoi"sv, "QOI image data", image_details)                               \
+    __ENUMERATE_MIME_TYPE_DESCRIPTION("text/markdown"sv, "Markdown document", description_only)                       \
+    __ENUMERATE_MIME_TYPE_DESCRIPTION("text/x-shellscript"sv, "POSIX shell script text executable", description_only)
+
+static Optional<DeprecatedString> get_description_from_mime_type(StringView const& mime, DeprecatedString const& path)
```
Comment (linusg) [inline Userland/Utilities/file.cpp]: Pass StringView by value (see 8b1108e)

File: Userland/Utilities/file.cpp [hunk 2]
```diff
@@ -19,13 +19,13 @@
...
 
-static Optional<DeprecatedString> description_only(DeprecatedString description, [[maybe_unused]] DeprecatedString const& path)
+static Optional<DeprecatedString> description_only(StringView description, [[maybe_unused]] DeprecatedString const& path)
 {
     return description;
 }
 
 // FIXME: Ideally Gfx::ImageDecoder could tell us the image type directly.
-static Optional<DeprecatedString> image_details(DeprecatedString const& description, DeprecatedString const& path)
+static Optional<DeprecatedString> image_details(StringView const& description, DeprecatedString const& path)
```
Comment (linusg) [inline Userland/Utilities/file.cpp]: Likewise

File: Userland/Utilities/file.cpp [hunk 3]
```diff
@@ -19,13 +19,13 @@
 #include <sys/stat.h>
 #include <unistd.h>
 
-static Optional<DeprecatedString> description_only(StringView description, [[maybe_unused]] DeprecatedString const& path)
+static Optional<DeprecatedString> description_only(StringView description, StringView const&)
```
Comment (linusg) [inline Userland/Utilities/file.cpp:22]: Likewise, and below

**Review by linusg — CHANGES_REQUESTED** [review]

File: Userland/Utilities/file.cpp [hunk 4]
```diff
@@ -19,13 +19,13 @@
...
 
-static Optional<DeprecatedString> description_only(StringView description, [[maybe_unused]] DeprecatedString const& path)
+static Optional<DeprecatedString> description_only(StringView description, StringView)
 {
     return description;
 }
 
 // FIXME: Ideally Gfx::ImageDecoder could tell us the image type directly.
-static Optional<DeprecatedString> image_details(StringView const& description, DeprecatedString const& path)
+static Optional<DeprecatedString> image_details(StringView description, StringView path)
```
Comment (linusg) [inline Userland/Utilities/file.cpp:28]: LHS `const&` removal belongs into the previous commit :)

**Comment by nico** [comment]
All done! I also hadn't run `git clang-format HEAD^` at each commit (or at any commit), which I've now fixed as well.

**Review by linusg — APPROVED** [review]

**Pull request merged**